### ISSUE TITLE
IM's create persistant tabs

### ISSUE THREAD

**Issue by CarlFK** [matrix-org/purple-matrix#59]
I am not really sure what proper behaviour is, but this bugs me:

When I IM someone, restart pidgin, it restores the connection into a new tab.  

I can't figure out how to get rid of them.

**Comment by flurrikat** [comment]
In the Buddy List, unchecking the "Persistent" item in the context menu of the chat(s) will cause it to not auto-join on startup.

As to why it happens-- it's intentionally set in matrix-sync.c. I'm not sure why that's the default, though? I personally feel that should be a setting.

**Comment by sleeksorrow** [comment]
I'm testing purple-matrix right now to see if matrix is suitable for me. Pidgin support would be mandatory in my scenario. And the first thing I stumbled upon was this one. It happens even if I disable "On reconnect, skip messages which were received in a previous session", and on the existing conversation I disable "Persistent" and "Auto-Join", but no change in behavior.

I guess when I want to use this plugin in my day-to-day scenario, this would mean that pidgin will open about 20 to 30 tabs at start. That would be awful.

How do others cope with this at the moment? Are there ways to mitigate this until solved? Or doesn't this happen with all conversations when there are multiple?

**Comment by ildar** [comment]
It doesn't happen every day, only on every Pidgin start. Hence it's
tolerable having this mess once every couple of weeks

**Comment by sleeksorrow** [comment]
Indeed. Unless you have a company policy that forces you to shut down the PC every evening ^^

**Comment by EionRobb** [comment]
https://github.com/kgraefe/pidgin-hide-chat-on-join is what I use to hide them

**Comment by sleeksorrow** [comment]
That's a nice workaround for sure, thank you! Also plays nice with another plugin that I use (which you wrote): pidgin-rocketchat :+1:
